src/components/Dialog.tsx

```diff
@@ -66,8 +66,6 @@ export function Provider({ value, children, options }: ProviderProps) {
   // If a Dialog is active, mark the main content inert
   const ref = useRef<HTMLDivElement>(null)
   const [active, setActive] = useState(false)
-  // If pageCentered dialogs are enabled, we should mount them directly to the document body
-  // and adjust the styling in the renderer below so that they are centered and sized correctly.
```

Comment:
confirming this comment isn't relevant any longer

Reply:
just wanted to move it, sorry for the confusion